eth/downloader/whitelist/milestone.go

```diff
@@ -66,6 +74,14 @@ func (m *milestone) Process(block uint64, hash common.Hash) {
 
 	m.finality.Process(block, hash)
 
+	for i := 0; i < len(m.FutureMilestoneOrder); i++ {
```

Comment:
is it just `for _, milestone := range m.FutureMilestoneOrder {`?

Reply:
Yes